### ISSUE TITLE
Image path

### ISSUE THREAD

**Issue by richasempire** [monsterhunters/Lora-Training-GUI#1]
Hi ,
It is really amazing to use this colab. I am facing errors for the images that I am adding for fine-tuning. I am putting them in dataset/images folder. However, it does recognise it.
here is the error:

22:53:25-519926 INFO     headless: True                                                             
22:53:25-733220 INFO     Using shell=True when running external commands...                         

UI finished loading, trying to launch (if it gets stuck here, it is having issues)

This is the URL to access UI: https://diamond-islamic-often-discussing.trycloudflare.com/                                |
IMPORTANT: You are using gradio version 4.26.0, however version 4.29.0 is available, please upgrade.
--------
Running on local URL:  http://127.0.0.1:7860/

To create a public link, set `share=True` in `launch()`.
22:56:08-188004 INFO     Start training LoRA Standard ...                                           
22:56:08-189534 INFO     Validating lr scheduler arguments...                                       
22:56:08-191634 INFO     Validating optimizer arguments...                                          
22:56:08-193306 INFO     Validating /content/kohya_ss/logs existence and writability... SUCCESS     
22:56:08-194579 INFO     Validating /content/kohya_ss/outputs existence and writability... SUCCESS  
22:56:08-196019 INFO     Validating runwayml/stable-diffusion-v1-5 existence... SKIPPING:           
                         huggingface.co model                                                       
22:56:08-197331 INFO     Validating /content/kohya_ss/dataset/images existence... SUCCESS           
22:56:08-198714 INFO     Headless mode, skipping verification if model already exist... if model    
                         already exist it will be overwritten...                                    
22:56:08-200376 INFO     Regulatization factor: 1                                                   
22:56:08-201562 INFO     Total steps: 0                                                             
22:56:08-202677 INFO     Train batch size: 1                                                        
22:56:08-203895 INFO     Gradient accumulation steps: 1                                             
22:56:08-205022 INFO     Epoch: 1                                                                   
22:56:08-206109 INFO     Max train steps: 1600                                                      
22:56:08-207479 INFO     stop_text_encoder_training = 0                                             
22:56:08-208750 INFO     lr_warmup_steps = 160                                                      
22:56:08-210750 INFO     Saving training config to                                                  
                         /content/kohya_ss/outputs/last_20240715-225608.json...                     
22:56:08-212702 INFO     Executing command: /usr/local/bin/accelerate launch --dynamo_backend no    
                         --dynamo_mode default --mixed_precision fp16 --num_processes 1             
                         --num_machines 1 --num_cpu_threads_per_process 2                           
                         /content/kohya_ss/sd-scripts/train_network.py --config_file                
                         /content/kohya_ss/outputs/config_lora-20240715-225608.toml                 
22:56:08-215136 INFO     Command executed.                                                          
2024-07-15 22:56:20 INFO     Loading settings from                                train_util.py:3744
                             /content/kohya_ss/outputs/config_lora-20240715-22560                   
                             8.toml...                                                              
                    INFO     /content/kohya_ss/outputs/config_lora-20240715-22560 train_util.py:3763
                             8                                                                      
2024-07-15 22:56:20 INFO     prepare tokenizer                                    train_util.py:4227
2024-07-15 22:56:21 INFO     update token length: 75                              train_util.py:4244
                    INFO     Using DreamBooth method.                           train_network.py:172
                    INFO     prepare images.                                      train_util.py:1572
                    INFO     0 train images with repeating.                       train_util.py:1613
                    INFO     0 reg images.                                        train_util.py:1616
                    WARNING  no regularization images /                           train_util.py:1621
                             正則化画像が見つかりませんでした                                       
                    INFO     [Dataset 0]                                          config_util.py:565
                               batch_size: 1                                                        
                               resolution: (512, 512)                                               
                               enable_bucket: False                                                 
                               network_multiplier: 1.0                                              
                                                                                                    
                                                                                                    
                    INFO     [Dataset 0]                                          config_util.py:571
                    INFO     loading image sizes.                                  train_util.py:853

0it [00:00, ?it/s]
0it [00:00, ?it/s]
                    INFO     make buckets                                          train_util.py:859
                    WARNING  min_bucket_reso and max_bucket_reso are ignored if    train_util.py:876
                             bucket_no_upscale is set, because bucket reso is                       
                             defined by image size automatically /                                  
                             bucket_no_upscaleが指定された場合は、bucketの解像度は                  
                             画像サイズから自動計算されるため、min_bucket_resoとma                  
                             x_bucket_resoは無視されます                                            
                    INFO     number of images (including repeats) /                train_util.py:905
                             各bucketの画像枚数（繰り返し回数を含む）                               
                    INFO     mean ar error (without repeats): nan                  train_util.py:915
                    ERROR    No data found. Please verify arguments             train_network.py:212
                             (train_data_dir must be the parent of folders with                     
                             images) /                                                              
                             画像がありません。引数指定を確認してください（trai                     
                             n_data_dirには画像があるフォルダではなく、画像があ                     
                             るフォルダの親フォルダを指定する必要があります）                       
22:56:23-908014 INFO     Training has ended.                                                        
22:59:47-854783 INFO     SD v2 model selected. Setting --v2 and --v_parameterization parameters     
23:00:14-453084 INFO     Start training LoRA Standard ...                                           
23:00:14-454372 INFO     Validating lr scheduler arguments...                                       
23:00:14-455806 INFO     Validating optimizer arguments...                                          
23:00:14-456997 INFO     Validating /content/kohya_ss/logs existence and writability... SUCCESS     
23:00:14-458178 INFO     Validating /content/kohya_ss/outputs existence and writability... SUCCESS  
23:00:14-459342 INFO     Validating stabilityai/stable-diffusion-2 existence... SKIPPING:           
                         huggingface.co model                                                       
23:00:14-460922 INFO     Validating ./dataset/images existence... SUCCESS                           
23:00:14-462267 INFO     Headless mode, skipping verification if model already exist... if model    
                         already exist it will be overwritten...                                    
23:00:14-464009 INFO     Regulatization factor: 1                                                   
23:00:14-465144 INFO     Total steps: 0                                                             
23:00:14-466234 INFO     Train batch size: 1                                                        
23:00:14-467327 INFO     Gradient accumulation steps: 1                                             
23:00:14-468419 INFO     Epoch: 1                                                                   
23:00:14-469472 INFO     Max train steps: 1600                                                      
23:00:14-470573 INFO     stop_text_encoder_training = 0                                             
23:00:14-471704 INFO     lr_warmup_steps = 160                                                      
23:00:14-473590 INFO     Saving training config to                                                  
                         /content/kohya_ss/outputs/last_20240715-230014.json...                     
23:00:14-475407 INFO     Executing command: /usr/local/bin/accelerate launch --dynamo_backend no    
                         --dynamo_mode default --mixed_precision fp16 --num_processes 1             
                         --num_machines 1 --num_cpu_threads_per_process 2                           
                         /content/kohya_ss/sd-scripts/train_network.py --config_file                
                         /content/kohya_ss/outputs/config_lora-20240715-230014.toml                 
23:00:14-477598 INFO     Command executed.                                                          
2024-07-15 23:00:26 INFO     Loading settings from                                train_util.py:3744
                             /content/kohya_ss/outputs/config_lora-20240715-23001                   
                             4.toml...                                                              
                    INFO     /content/kohya_ss/outputs/config_lora-20240715-23001 train_util.py:3763
                             4                                                                      
                    WARNING  v2 with clip_skip will be unexpected /               train_util.py:3451
                             v2でclip_skipを使用することは想定されていません                        
2024-07-15 23:00:26 INFO     prepare tokenizer                                    train_util.py:4227
2024-07-15 23:00:29 INFO     update token length: 75                              train_util.py:4244
                    INFO     Using DreamBooth method.                           train_network.py:172
                    INFO     prepare images.                                      train_util.py:1572
                    INFO     0 train images with repeating.                       train_util.py:1613
                    INFO     0 reg images.                                        train_util.py:1616
                    WARNING  no regularization images /                           train_util.py:1621
                             正則化画像が見つかりませんでした                                       
                    INFO     [Dataset 0]                                          config_util.py:565
                               batch_size: 1                                                        
                               resolution: (512, 512)                                               
                               enable_bucket: False                                                 
                               network_multiplier: 1.0                                              
                                                                                                    
                                                                                                    
                    INFO     [Dataset 0]                                          config_util.py:571
                    INFO     loading image sizes.                                  train_util.py:853

0it [00:00, ?it/s]
0it [00:00, ?it/s]
                    INFO     make buckets                                          train_util.py:859
                    WARNING  min_bucket_reso and max_bucket_reso are ignored if    train_util.py:876
                             bucket_no_upscale is set, because bucket reso is                       
                             defined by image size automatically /                                  
                             bucket_no_upscaleが指定された場合は、bucketの解像度は                  
                             画像サイズから自動計算されるため、min_bucket_resoとma                  
                             x_bucket_resoは無視されます                                            
                    INFO     number of images (including repeats) /                train_util.py:905
                             各bucketの画像枚数（繰り返し回数を含む）                               
                    INFO     mean ar error (without repeats): nan                  train_util.py:915
                    ERROR    No data found. Please verify arguments             train_network.py:212
                             (train_data_dir must be the parent of folders with                     
                             images) /                                                              
                             画像がありません。引数指定を確認してください（trai                     
                             n_data_dirには画像があるフォルダではなく、画像があ                     
                             るフォルダの親フォルダを指定する必要があります）                       
23:00:32-528989 INFO     Training has ended.




Your help will be valuable. Thank you!!

**Comment by monsterhunters** [comment]
@richasempire , train_data_dir must be the parent of folders with
images
